TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.CfgSentence.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using TypeCobol.Compiler.Symbols;
+
+namespace TypeCobol.Analysis.Cfg
+{
+    public partial class ControlFlowGraphBuilder<D>
+    {
+        /// <summary>
+        /// A Sentence used for our builder. A Sentence is a special symbol.
+        /// </summary>
+        internal class CfgSentence : Symbol
```

Comment:
A sentence looks like more a block group than a symbol.

Reply:
This is a sentence from a COBOL perspective, and CfgSectionSymbol and CfgParagraphSymbol contains sentences seen as symbols. It is a convenience to see a Sentence as Symbol so that it can be given an Owner which is its Section or Paragraph symbol.

Reply:
See PR #1738, Sentence keeps its list of blocks.